Pagination

Pwma77 opened the pull request:
Lemme know how I performed this time. I know it was kinda tricky due of what happened with PyCharm and stuff. Got way too stuck and frustrated about it, but thanks to you I could do it xD
Still, if there some stuff weird-looking or sth, tell 'em plz.

Pwma77 committed:
Pagination working - two inputs x little fixes & no tests checked

Pwma77 committed:
Pagination tests with no errors

BrandomRobor reviewed (changes requested):
You need to combine the schemas first, we'll see it today

Pwma77 committed:
Pagination and filters one input fix

Pwma77 committed:
Pagination and filters unit test fix

BrandomRobor committed:
Added filter to prevent pagination options to count as property filters

BrandomRobor committed:
Removed redundant integer conversions

BrandomRobor committed:
Fixed property tests
The controller expects the correct types, validation and conversion is done within the routes

BrandomRobor reviewed (approved):
CI keeps failing due to a strange bug, but I confirm everything works as intended
